WooCommerce/Classes/ViewRelated/Products/Edit Product/Linked Products List Selector/LinkedProductsListSelectorViewController.swift

```diff
@@ -29,7 +29,7 @@ final class LinkedProductsListSelectorViewController: UIViewController {
             return PaginatedListSelectorViewController(viewProperties: viewProperties, dataSource: dataSource, onDismiss: { _ in })
     }()
 
-    private var cancellable: ObservationToken?
+    private var cancellable: AnyCancellable?
```

Comment:
Nit: should this be renamed to `productIDsSubscription`?

Reply:
renamed in https://github.com/woocommerce/woocommerce-ios/pull/5988/commits/1482763e5c5dc81e4d1926826cbe7e0cf0fe62f4